mediator/esmFldsExchange_ufs_mod.F90

```diff
@@ -68,15 +68,15 @@ subroutine esmFldsExchange_nems(gcomp, phase, rc)
     if (chkerr(rc,__LINE__,u_FILE_u)) return
 
     ! Set maptype according to coupling_mode
-    if (trim(coupling_mode) == 'nems_orig' .or. trim(coupling_mode) == 'nems_orig_data') then
+    if (trim(coupling_mode) == 'ufs.nfrac' .or. trim(coupling_mode) == 'ufs.nfrac.aoflux') then
       maptype = mapnstod_consf
```

Comment:
Can you double check these.  I see ufs.nfrac, ufs.nfrac.aoflux and ufs.frac.aoflux - are all of these correct?